index.js

```diff
@@ -0,0 +1,13 @@
+let divide = (a, b) => {
+  let reuslt = a / b;
+  return reuslt;
+}
```

Comment:
What you have works, which is awesome, but you can also write it like this, just like you have in the one below:
```suggestion
const divide = (a, b) => a / b;
```